src/main/java/seedu/agendum/model/task/ReadOnlyTask.java

```diff
@@ -9,6 +11,9 @@
 public interface ReadOnlyTask {
 
     Name getName();
+    boolean isCompleted();
+    LocalDateTime getStartDateTime();
+    LocalDateTime getEndDateTime();
```

Comment:
In order to support Optionals start by changing this to:

     Optional<LocalDateTime> getStartDateTime();
     Optional<LocalDateTime> getEndDateTime();